### PR TITLE
Fix/quick

### PR DESCRIPTION
**Pull request by wilmveel**

### CODE REVIEW

File: src/main/kotlin/controllers/PersonController.kt [hunk 1]
```diff
@@ -23,6 +22,7 @@ import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import org.springframework.web.server.ResponseStatusException
+import java.security.Principal
```
Comment (tkgroot) [inline src/main/kotlin/controllers/PersonController.kt]: The import order matters here. The previous implementation was the correct lexicographic order
   [ktlint] /Users/tkgroot/projects/flock/flock-eco-holidays/src/main/kotlin/controllers/PersonController.kt
   [ktlint]   3:1 Imports must be ordered in lexicographic order without any empty lines in-between
   [ktlint] "checkstyle" report written to /Users/tkgroot/projects/flock/flock-eco-holidays/ktlint.xml

File: src/main/kotlin/model/Person.kt [hunk 2]
```diff
@@ -6,9 +6,13 @@ import com.fasterxml.jackson.annotation.ObjectIdGenerators
 import community.flock.eco.core.events.EventEntityListeners
 import community.flock.eco.core.model.AbstractCodeEntity
 import community.flock.eco.feature.user.model.User
+import org.hibernate.annotations.OnDelete
+import org.hibernate.annotations.OnDeleteAction
 import java.util.UUID
+import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.EntityListeners
+import javax.persistence.ForeignKey
```
Comment (tkgroot) [inline src/main/kotlin/model/Person.kt]: unused imports

File: src/main/kotlin/controllers/PersonController.kt [hunk 3]
```diff
@@ -38,40 +38,34 @@ class PersonController(
...
             val userCode = when {
-                it.isAdmin() -> form.userCode
-                else -> it.code
+                user.isAdmin() -> null
+                else -> user.code
             }
             form.copy(userCode = userCode)
-
-            return@let service.create(form)
+                .let { service.create(it) }
```
Comment (tkgroot) [inline src/main/kotlin/controllers/PersonController.kt]: is there an advantage doing it this way or just preference? 
Im very much in favor of consistency.
(my personal dislike of curly brackets and thus favoring `return@let`)

Comment (wilmveel) [reply src/main/kotlin/controllers/PersonController.kt]: I'm not sure if both give the same output. My preference is to write in a more functional style. I will check the return@let syntax I have never seen it before.

Comment (tkgroot) [reply src/main/kotlin/controllers/PersonController.kt]: dont look too far ;) https://kotlinlang.org/docs/reference/returns.html

**Pull request merged**